UtopiaCity/Data/Migrations/20210804064209_Update_IdentityUser_V1.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace UtopiaCity.Data.Migrations
+{
+    public partial class Update_IdentityUser_V1 : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropTable(
+                name: "CitizenAccUsers");
+
+            migrationBuilder.AddColumn<double>(
+                name: "Balance",
+                table: "AspNetUsers",
+                nullable: true);
+
+            migrationBuilder.AddColumn<string>(
+                name: "Gender",
+                table: "AspNetUsers",
+                nullable: true);
+
+            migrationBuilder.AddColumn<string>(
+                name: "Name",
+                table: "AspNetUsers",
+                nullable: true);
+
+            migrationBuilder.AddColumn<string>(
+                name: "Surname",
+                table: "AspNetUsers",
+                nullable: true);
+        }
```

Comment:
Finally, you added that fields to IdentityUser, so no need for AppUser model because you can access them in IdentityUser.